### PR TITLE
Feature/si 2658 image uri

### PR DESCRIPTION
**Pull request by zer0stars**
- Index ImageURI attribute from the chain ```https://polygonscan.com/tx/0xc10fce2b6c7ae9367d07d0fc0f9809bf97db6215d3513cad3ad7e874c50af409#eventlog```
- Update image to imageURI in response
- Add imageUri column to vehicle tables

### CODE REVIEW

**Comment by zer0stars** [comment]
@LorranSutter did we make a mistake here by doing ImageUri instead of ImageURI in the contract attributes? It hasn't been used so I think we can change it if needed.

**Comment by elffjs** [comment]
We’re in TypeScript/Solidity land. I think `ImageUri` fits okay.

**Comment by LorranSutter** [comment]
Well... it is kinda out of the pattern we've been following. We have `DataURI`, `DefinitionURI`, but `ImageUri`. It hurts me a bit. We can change.

**Comment by elffjs** [comment]
That I didn't see. Yeah, we'd better change then.

File: graph/schema/vehicle.graphqls [hunk 1]
```diff
@@ -126,7 +126,7 @@ type Vehicle implements Node {
   """
   The Image Url of he vehicle
   """
-  image: String!
+  imageUri: String!
```
Comment (elffjs) [inline graph/schema/vehicle.graphqls:129]: Didn't catch that the first time.

File: internal/repositories/aftermarket/aftermarket_devices.go [hunk 2]
```diff
@@ -191,7 +191,7 @@ type aftermarketDevicePrimaryKey struct {
 	TokenID int
 }
 
-// ToAPI converts a vehicle to a corresponding API vehicle.
+// ToAPI converts a aftermarket device to a corresponding API aftermarket device.
```
Comment (elffjs) [inline internal/repositories/aftermarket/aftermarket_devices.go]: I'm going to fix the grammar here.

**Comment by elffjs** [comment]
Rewrote some of this in #100. If you're willing to go further, in #101 we get rid of this URI construction function returning errors and things clean up a little.

**Review by elffjs — COMMENTED** [review]
This will be poggers when it compiles.

File: internal/repositories/vehicle/vehicles.go [hunk 3]
```diff
@@ -42,11 +42,16 @@ func (r *Repository) createVehiclesResponse(totalCount int64, vehicles models.Ve
 	nodes := make([]*gmodel.Vehicle, len(vehicles))
 
 	for i, dv := range vehicles {
-		imageURL, err := GetVehicleImageURL(r.Settings.BaseImageURL, dv.ID)
```
Comment (elffjs) [inline internal/repositories/vehicle/vehicles.go:45]: We need to do this for the singular get too.

**Comment by elffjs** [comment]
Test failure appears to be some Testcontainers thing. I think this does what you want now.

**Review by KevinJoiner — APPROVED** [review]
tests passed after a re-run

**Review by elffjs — APPROVED** [review]

**Pull request merged**